### ISSUE TITLE
Cant Build on Android

### ISSUE THREAD

**Issue by francorobles** [NativeScript/push-plugin#140]
Please fix the version conflict either by updating the version of the google-services plugin (information about the latest version is available at https://bintray.com/android/android-tools/com.google.gms.google-services/) or updating the version of com.google.android.gms to 10.0.1

**Comment by jogboms** [comment]
Any luck with this?

**Comment by TheOnlyMatt** [comment]
Same problem here, even after updating Google Play Services with Android Studio's SDK Manager.
I also ran `sdkmanager --update`

But stills getting the same error : 
`Execution failed for task ':processF0F1F2F3F4F5F6F7F8DebugGoogleServices'.
> Please fix the version conflict either by updating the version of the google-services plugin (information about the latest version is available at https://bintray.com/android/android-tools/com.google.gms.google-services/) or updating the version of com.google.android.gms to 11.0.+.
`

**Comment by TheOnlyMatt** [comment]
Finally succeeded to build my app and receive notifications without crashing.

In the root level of /app/App_Resources/Android/app.gradle, I added the firebaseMessagingVersion : 
```
project.ext {
    ...
    firebaseMessagingVersion = "11.0.+" // OR the version you wish
}
```
You also have to delete from the AndroidManifest, <Activity>, <Service> and <Receiver> tags.
Then in build.gradle I have 
`dependencies {
		classpath "com.google.gms:google-services:+"
		classpath "com.android.tools.build:gradle:2.2.3"
                classpath "com.google.gms:google-services:3.1.0"
    }`

This worked for me

**Comment by lini** [comment]
This issue should be fixed in version 0.1.5+ of the plugin.
